### ISSUE TITLE
SD Card Support 

### ISSUE THREAD

**Issue by Metalomeus** [flyinghead/flycast#1710]
Has SD card support for flycast data been dropped.....2.3.2 211 had support for it.....it comes in handy because of custom texture packs etc..

Regards

**Comment by flyinghead** [comment]
Yes, the home folder is no longer configurable. This is due to Google's mandatory requirements to be published in Google Play.

**Comment by Metalomeus** [comment]
Ok .. that sucks... Is it possible to accomplish that via ci or GitHub
releases?

flyinghead ***@***.***> schrieb am Do., 24. Okt. 2024, 16:08:
